modules/jsonrpc/register_types.cpp

```diff
@@ -34,6 +34,10 @@
 
 #include "core/object/class_db.h"
 
+#ifdef TESTS_ENABLED
+#include "tests/test_jsonrpc.h"
+#endif
```

Comment:
I think this part can be removed. Other modules like `modules/noise` don't include their own test files like this, and they still get run.

Reply:
Thank you so much - that seems to have addressed this!